src/game/hud.cpp

```diff
@@ -39,6 +40,7 @@ namespace hud
     VAR(IDF_PERSIST, showloadingversion, 0, 1, 1);
     VAR(IDF_PERSIST, showloadingurl, 0, 1, 1);
 
+    VAR(IDF_PERSIST, showsystemtime, 0, 0, 1);
```

Comment:
Aren't HUD options prefixed `hud` or so?

Reply:
nope, they're not